references/organization/_06-Project-Plan-and-Workflow.md

```diff
@@ -0,0 +1,64 @@
+    - One where all the Xs are independent (i.e. the only edges in the causal graph are between the X and the outcome variable). 
+    - One based on a realistic causal graph with confounders. Preferably focus on a case with one latent confounder (like distance? Age? Gender?) <br>
+-	For each of the two causal graphs above, simulate choice data based on the outcome model assumed. <br>
+-	Estimate the choice model for each of those two datasets. <br>
+-	Repeat the process n times, and calculate the percentage of those times where the estimated parameters’ confidence intervals contain the true parameter. <br>
+
+### Dealing with latent confounders in the causal graph
+
+In this part of the work, we take the realistic causal graph from part 1 and remove a latent confounder from it. We then attempt to recover the latent confounder using the de-confounder algorithm proposed by Blei et al. (2018). We will apply this work to the simulated data set, where we know the true parameters of interest. We will repeat the same simulation process in part 1 where we sample from the population and estimate models where we control for the latent confounder using the deconfounder approach and look at the percent of time we recover the true parameter. 
+
+##### Concrete steps: 
+
+Functions <br>
+-	A class with functions to estimate factor models. Start with PPCA as in the deconfounder’s tutorial and expand as needed. <br>
+-	A function to perform posterior predictive checks on the factor model to assess its predictive accuracy. <br>
+
+Workflow <br>
+-	Start with a hypothesized general causal graph. For the simulated example, we’ll start by assuming the true causal graph used to generate the data, where we highlight the confounder as being latent/unobserved. <br>
+-	Then, come up with a “utility level” causal graph, where we come up with a causal graph for each utility equation since the confounding is at the level of the utilities in a MNL context. These causal graphs would inform the utility specifications, and will determine which utilities do and do not require estimating a latent confounder. We discussed this briefly before, but are we on the same page? <br>
+-	Based on the above specifications, estimate the required latent confounders using a factor model, start with PPCA. <br>
+-	Perform posterior predictive checks on the estimated factor model and assess goodness of fit. Since we’re doing this on a simulated dataset, then we know in advance how many latent confounders there are. We could potentially try controlling for more confounders and see how that affects the results (in other words, see how sensitive our analysis is to the number of confounders used). <br>
+-	Once the checks pass, include the latent confounder in the utility equations, and estimate a MNL. <br>
+-	Repeat the process n times, and calculate the percentage of those times where the estimated parameters’ confidence intervals contain the true parameter. <br>
```

Comment:
I would only do the visual posterior predictive checking once (as opposed to n times). These take some time to interpret visually and you def won't be doing 100s of these.

You should leaving numeric posterior predictive checks in the code though. The idea would be to have a statement like `if not min_p_value < posterior_predictive_p_value and posterior_predictive_p_value < max_p_value: raise ModelValidationError`.